docs/topics_5/web-execute.adoc

```diff
@@ -3,6 +3,21 @@
 [id='execute_{context}']
 = Running an analysis
 
-From the analysis configuration page, click the *Save & Run* button to execute the analysis.
+You can run an analysis from the *Review* page of the *Create project* interface or from the *Analysis Configuration* page.
 
-On the *Analysis* page, the progress of the analysis that is currently executing is shown. Completed, queued, and in-progress executions are listed in the table. Click the *Run Analysis* to run the analysis again with the latest configuration.
+.Procedure
+
+. Click *Save and run*.
```

Comment:
This is not how a procedure begins. You can't put the info about where to begin in the introduction.

Module/procedure must stand alone, like this:

In the web console, click Analysis (or Analysis > Configuration or however you get to it).
Click Save and Run.
etc.

Don't add useless notes saying "you can do x by opening y". That's a procedure and probably goes somewhere else.
On the other hand, if it's just another way of doing the same thing, drop the note altogether.